identity-iota/src/error.rs

```diff
@@ -53,4 +53,8 @@ pub enum Error {
   NoExplorerURLSet,
   #[error("Invalid Explorer Url")]
   InvalidExplorerURL,
+  #[error("compression error")]
+  CompressionError,
+  #[error("invalid message flags")]
+  InvalidMessageFlags,
```

Comment:
Do we want to be more specific with `InvalidMessageVersion` and `InvalidMessageEncoding`?

Reply:
Since there's too many Errors already, and they probably will get refactored anyway, maybe we can leave it as is for now.